staging/src/k8s.io/client-go/tools/cache/delta_fifo.go

```diff
@@ -161,12 +161,14 @@ type DeltaFIFO struct {
 	lock sync.RWMutex
 	cond sync.Cond
 
+	// We maintain the properties that:
+	// - a key is in `items` if and only if that key is in `queue`;
+	// - a given key appears in at most one position in `queue`;
```

Comment:
What's your thoughts of simplifying this statement to `There's no duplicates in queue`. 
When I see wordings like `at most`, I always think of something QoS related, i.e. "at most once", "at least once", "exactly once" etc.

Reply:
Reworded.